bliss/encoder/image_normalizer.py

```diff
@@ -138,13 +108,7 @@ def get_input_tensor(self, batch):
             if self.include_background:
                 inputs[0] = self.clahe(backgrounds, self.clahe_min_stdev)
 
-        if self.asinh_params:
-            processed_raw_images = (raw_images - self.asinh_thresholds_tensor) * torch.exp(
-                self.asinh_scales_tensor
-            )
-            inputs.append(torch.asinh(processed_raw_images))
```

Comment:
It would be nice if everyone had access to the asinh normalization eventually. But perhaps it's still too experimental?